webpack.config.js

```diff
@@ -20,7 +20,27 @@ module.exports = {
       },
       {
         test: /\.scss$/,
-        use: ["style-loader", "css-loader", "sass-loader"]
+        use: [
+          {
+            loader: "style-loader"
+          },
+          {
+            loader: "css-loader"
+          },
+          {
+            loader: "sass-loader",
+            options: {
+              importer: function(url, prev) {
```

Comment:
you do not need this importer you can import in scss from node modules with just

```scss
@import "~@material/layout-grid/mdc-layout-grid";
```
notice tilda